sechub-administration/src/test/java/com/mercedesbenz/sechub/domain/administration/UserRepositoryDBTest.java

```diff
@@ -132,7 +114,28 @@ public void user_NOT_being_owner_of_project_but_listed_as_user_can_be_deleted__a
         assertProjectFound(project);
 
         assertUserNotFound(user1);
+    }
 
+    @Test
+    void existsByEmailIgnoreCase_returns_false_when_email_not_in_database() {
+        assertThat(userRepository.existsByEmailIgnoreCase("very_new_email@mail.com")).isFalse();
+    }
+
+    @Test
+    void existsByEmailIgnoreCase_returns_true_when_email_exists_in_database() {
+        /* prepare */
+        String mail1 = user1.getEmailAddress();
+        String mail2 = user2.getEmailAddress();
+        String mail3 = mail1.toUpperCase();
+        String mail4 = mail2.toUpperCase();
+        String mail5 = mail1.replace("example", "exaMplE");
```

Comment:
I think it is a good way to change the existing mail address to a case randomized variant. But please add following here

```java
/* check preconditions */
assertThat(mail5.contains("exaMplE")).isTrue(); // ensure example mail address is as expected...

```
why ? You rely on the situation that the test helper class created a mail address containing always "example" inside but... somebody could change the Test helper class... 
So it is necessary to check the precondition.